tests/playTests.sh

```diff
@@ -258,7 +259,7 @@ rm ./*.tmp ./*.zstd
 $ECHO "frame concatenation tests completed"
 
 
-if [ "$isWindows" = false ] && [ "$UNAME" != 'SunOS' ] ; then
+if [ "$isWindows" = false ] && [ "$UNAME" != 'SunOS' ] && [ "$(uname -s)" != "OpenBSD" ] ; then
```

Comment:
Does plain `$(uname)` not default to `-s` on OpenBSD? If it does, please use `$UNAME`.